src/hooks/core/websocket/types.d.ts

```diff
@@ -90,6 +90,8 @@ export type GetstreamChannel = {
   last_message_at: string;
   member_count: number;
   members: any[];
+  better_channel_members: any[];
+  better_channel_member: any[];
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new properties `better_channel_members` and `better_channel_member` are of type `any[]`. Using `any` in TypeScript defeats the purpose of using TypeScript, which is to have static types. It would be better if you could define an interface for these properties to provide a clear understanding of what they should contain.

```diff
-  better_channel_members: any[];
-  better_channel_member: any[];
+  better_channel_members: BetterChannelMember[];
+  better_channel_member: BetterChannelMember[];
```
Here, `BetterChannelMember` is a hypothetical interface that you should replace with the actual structure expected in these arrays.



<!-- This is an auto-generated comment by OSS CodeRabbit -->